_includes/getting-started.html

```diff
@@ -2,10 +2,10 @@
     <div style="margin-top: auto;" id="box"> 
       <h1 style="text-align: left;margin-left: 80px;"><strong style="font-size: 90px;">Get Started</strong><br>
         with Meshery
-      <p style="font-size: 20px;">Choose a platform and see how easy it is to get up and running with a service mesh.</p></h1>
+      <p style="font-size: 20px;">Choose a platform and see how easy<br> it is to get up and running with a service mesh.</p></h1>
     </div>
   </div> 
-<div class="container flex" id="containers" style="margin-left: 680px;margin-top: -250px;">
+<div class="container flex" id="containers" style="margin-left: 680px;margin-top: -210px;">
```

Comment:
You'll want to work toward a more dynamic approach than specifying in pixels. Resources:

- https://webdesign.tutsplus.com/tutorials/a-comprehensive-guide-to-flexbox-sizing--cms-31948
- https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
@leecalcote sir, now I have solved this issue